app/models/miq_product_feature.rb

```diff
@@ -169,21 +165,30 @@ def self.seed_tenant_miq_product_features
     Tenant.in_my_region.all.flat_map { |t| seed_single_tenant_miq_product_features(t) }
   end
 
-  def self.seed_features(path = FIXTURE_PATH)
-    fixture_yaml = feature_yaml(path)
+  def self.seed_features
+    transaction do
+      features = all.index_by(&:identifier)
+
+      root_file, other_files = seed_files
 
-    features = all.to_a.index_by(&:identifier)
-    seen     = seed_from_hash(YAML.load_file(fixture_yaml), seen, nil, features)
+      seen = seed_from_hash(YAML.load_file(root_file), seen, nil, features)
+      root_feature = find_by(:identifier => SUPER_ADMIN_FEATURE)
 
-    root_feature = MiqProductFeature.find_by(:identifier => SUPER_ADMIN_FEATURE)
-    Dir.glob(path.join("*.yml")).each do |fixture|
-      seed_from_hash(YAML.load_file(fixture), seen, root_feature)
+      other_files.each do |file|
+        seed_from_array(YAML.load_file(file), seen, root_feature)
```

Comment:
Let me say this to make sure I understand.
1) seed_from_hash is used for the core product features where the "everything" identifier is the root feature and all children fall under it

2) seed_from_array is for seeding an array of features all at the same level:  direct children of the root "everything" feature.  Each can have their own trees of features but they all get the "everything" feature as the root.

Reply:
Kind of?  I didn't want to add the seed_from_array, but plugins can bring an array of features, so it made sense to abstract walking an array of hashes.